GoogleDataTransport/GDTCORLibrary/GDTCORPlatform.m

```diff
@@ -124,6 +130,16 @@ - (BOOL)isAppExtension {
 #endif
 }
 
+- (BOOL)isRunningInBackground {
+#if TARGET_OS_IOS || TARGET_OS_TV
```

Comment:
Where should Catalyst be?

Reply:
Good catch! Should be with macOS. Removed the `elif` in favour of `else`